buildkite/src/Jobs/Lint/Rust.dhall

```diff
@@ -24,7 +24,7 @@ Pipeline.build
     steps = [
       Command.build
         Command.Config::{
-          commands = [ Cmd.runInDocker jobDocker "cd src/app/trace-tool ; cargo check --frozen" ]
+          commands = RunInToolchain.runInToolchainBuster ([] : List Text) "cd src/app/trace-tool ; cargo check --frozen"
```

Comment:
You'll probably need to update your path with `export PATH=/home/opam/.cargo/bin:$PATH` to make this work.